docs/guides/index.markdown

```diff
@@ -3,22 +3,36 @@ layout: basic
 title: Guides
 ---
 
-### Guides
+## Guides
 
-#### User Guides
+### User Guides
 
 * [AeroGear Android User Guide](aerogear-android)
 * [AeroGear iOS 1.X User Guide](aerogear-ios)
 * [AeroGear iOS 2.X User Guide](aerogear-ios-2.X)
-* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
-* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-#### UnifiedPush Server
+### UnifiedPush Server
 
 * [AeroGear UnifiedPush Server resources](../unifiedpush)
 
+### Security
 
-#### Contributing to AeroGear
+* OTP
+  * [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+  * [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
+
+* OAuth2
+  1. [Basic steps](security/oauth2-basic-steps)
```

Comment:
@lfryc that's the idea, be general, `various OAuth2 providers` is general in the same way. Most of the guides outside make use of this sentence.